From 27a0e2f9eb2f01a8f0ec2dc783be56962aa78f4c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20Kotyla?= <michal.kotyla@3mdeb.com>
Date: Thu, 24 Nov 2022 13:10:08 +0100
Subject: [PATCH] hwdb: add Clevo touchpad toggle key quirks

(cherry picked from commit cb8d22fa8490ea19e82117b1e1ea794e3c0a4c98)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 498a4c5f5e..9707132bee 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -278,6 +278,13 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnNotebook:pnW65_67SZ:*
  KEYBOARD_KEY_ae=!volumedown
  KEYBOARD_KEY_b0=!volumeup
 
+evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNS50_70MU:*
+evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNV4XMB,ME,MZ:*
+evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNS5x_NS7xPU:*
+evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNV4xPZ:*
+ KEYBOARD_KEY_f7=f21                                    # Touchpad Toggle
+ KEYBOARD_KEY_f8=f21                                    # Touchpad Toggle
+
 ###########################################################
 # Compal
 ###########################################################
